test/test_boundary_conditions.jl

```diff
@@ -45,8 +56,13 @@ end
 function test_flux_budget(arch, TF, field_name)
     N, κ, Lz = 16, 1, 0.7
 
+    bottom_flux = TF(0.3)
+    flux_bc = BoundaryCondition(Flux, bottom_flux)
+    modelbcs = BoundaryConditions(field_name, :z, :bottom, flux_bc)
```

Comment:
@ali-ramadhan see here for how to create `BoundaryConditions` with one non-default boundary conditions.